packages/flutter/lib/src/material/navigation_bar.dart

```diff
@@ -111,6 +112,13 @@ class NavigationBar extends StatelessWidget {
   /// [ColorScheme.onSurface] using an [ElevationOverlay].
   final Color? backgroundColor;
 
+  /// The elevation of the [NavigationBar] itself.
+  ///
+  /// If null, [NavigationBarThemeData.elevation] is used. If that
+  /// is also null, then if [ThemeData.useMaterial3] is true then it will
+  /// be 3.0 otherwise 0.0.
```

Comment:
When we were working on this with designers, I believe this component (and the app bar) were supposed to have no shadow but still receive a surface overlay. It appears the tokens do not indicate this though.

I've messaged designers to see what the deal is exactly and I'll CC you Darren.

Reply:
Thx. It would be nice to have this somehow encoded in the tokens. We need the elevation for the overlay calculation, so it might make sense to have a `useShadow` or such in the tokens that we would mirror in the `Material` widget.

Reply:
According to our designer, the current way this is indicated is that a `.shadow-color` token is present when there should be a shadow. However for the navigation bar, it is present but that is a bug. So that should get fixed in another version of the tokens. It's a bit of hacky logic however

Reply:
What about just changing the value of the shadow color to transparent for the navigationbar? That would be way less hacky I think.